app/src/main/java/com/sdp13epfl2021/projmag/database/di/DatabaseModules.kt

```diff
@@ -0,0 +1,33 @@
+package com.sdp13epfl2021.projmag.database.di
+
+import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.firestore.FirebaseFirestore
+import com.sdp13epfl2021.projmag.database.impl.firebase.FirebaseUserdataDatabase
+import com.sdp13epfl2021.projmag.database.interfaces.UserdataDatabase
+import dagger.Binds
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+abstract class UserdataDatabaseModule {
+    @Binds
+    @Singleton
+    abstract fun bindUserdataDatabase(
+        userDB: FirebaseUserdataDatabase
+    ): UserdataDatabase
```

Comment:
Is it possible to use the OfflineCachedUserdataDatabase implementation directly ?
If not, is it because it needs a directory and the localUserID ?

Reply:
Yes I can I did it in my other PR. I will change it before mergin